Formula/gcc.rb

```diff
@@ -111,23 +111,19 @@ def install
       args << "--build=#{cpu}-apple-darwin#{OS.kernel_version.major}"
       args << "--with-system-zlib"
 
-      # Xcode 10 dropped 32-bit support
-      args << "--disable-multilib" if DevelopmentTools.clang_build_version >= 1000
+      # Xcode 10 dropped 32-bit Intel support
+      args << "--disable-multilib" if Hardware::CPU.intel? && DevelopmentTools.clang_build_version >= 1000
 
-      # Workaround for Xcode 12.5 bug on Intel
+      # Workaround for Xcode 12.5 bug on Intel, remove in next version
       # https://gcc.gnu.org/bugzilla/show_bug.cgi?id=100340
-      args << "--without-build-config" if Hardware::CPU.intel? && DevelopmentTools.clang_build_version >= 1205
+      args << "--without-build-config" if Hardware::CPU.intel? && DevelopmentTools.clang_build_version == 1205
 
       # System headers may not be in /usr/include
       sdk = MacOS.sdk_path_if_needed
       if sdk
         args << "--with-native-system-header-dir=/usr/include"
         args << "--with-sysroot=#{sdk}"
       end
-
-      # Ensure correct install names when linking against libgcc_s;
-      # see discussion in https://github.com/Homebrew/legacy-homebrew/pull/34303
-      inreplace "libgcc/config/t-slibgcc-darwin", "@shlib_slibdir@", "#{HOMEBREW_PREFIX}/lib/gcc/#{version_suffix}"
```

Comment:
Is `@shlib_slibdir@` replaced with `@rpath` now?

Reply:
The compiler should now be fully relocatable, and the path to `libgcc_s` should not be recorded. I want to check that, and download the fresh bottles, once they have been built. Is it possible while some of the job are still running?

Reply:
AFAIK, no. 😞 I'll have a look too when CI finishes; it shouldn't be too long now. 

It's nice for this to have been fixed, since it's a bit silly for the compiler itself to be completely relocatable but the binaries it spits out to remember stuff that's true only when the compiler was built.

Reply:
In theory, the binaries produced are made relocatable as well (since a couple of versions). If it's not the case, we need to identify the issue and report it, because it can be fixed for sure.